Write contributing guidelines issue

S-S-Poojitha opened the pull request:
I have added the contributing guidelines

S-S-Poojitha committed:
Add files via upload

S-S-Poojitha committed:
Update README.md

hagenderouen commented:
@S-S-Poojitha Thank you for your contribution. This is a great start. The contributing guidelines are repetitive of the README. Please remove the repetitive information from the README with a short statement and link to the contributing guidelines. Updating repo references to `https://github.com/hagenderouen/awesome-readme.git` and links to the contributing guidelines document. Please remove the code of conduct link since it wasn't added.

S-S-Poojitha committed:
Update README.md

S-S-Poojitha committed:
Update README.md

S-S-Poojitha committed:
Update README.md

S-S-Poojitha committed:
Update Contributing_Guidelines.md

S-S-Poojitha committed:
Update README.md

S-S-Poojitha committed:
Update Contributing_Guidelines.md

S-S-Poojitha committed:
Update Contributing_Guidelines.md

S-S-Poojitha committed:
Update README.md

S-S-Poojitha committed:
Update README.md

S-S-Poojitha commented:
Thank you for your feedback! I have made the following changes as requested:
Removed the repetitive information from the README and added a short statement linking to the contributing guidelines.
Updated the repository references to https://github.com/hagenderouen/awesome-readme.git.
Removed the code of conduct link as it wasn't added.

hagenderouen commented on Contributing_Guidelines.md:
````diff
@@ -0,0 +1,83 @@
+3. **Set Upstream Remote**
+
+   ```bash
+   git remote add upstream https://github.com/original-owner/repository-name.git
````
Update repo url to `https://github.com/hagenderouen/awesome-readme.git`

hagenderouen commented:
@S-S-Poojitha Please update the git repo URL to `https://github.com/hagenderouen/awesome-readme.git` in contributing guidelines. Merge the latest changes from `README.md`.

S-S-Poojitha committed:
Update Contributing_Guidelines.md

S-S-Poojitha committed:
Update README.md

S-S-Poojitha committed:
Update README.md

S-S-Poojitha committed:
Update README.md
